server.js

```diff
@@ -1,26 +1,24 @@
-const express = require('express');
-const bodyParser = require('body-parser');
-const helmet = require('helmet');
-const compression = require('compression');
+import express from 'express';
+import 'airbnb-browser-shims';
+import bodyParser from 'body-parser';
+import helmet from 'helmet';
+import compression from 'compression';
 
-const users = require('./routes/users');
-const questions = require('./routes/questions');
-const rsvp = require('./routes/rsvp');
-const meetups = require('./routes/meetups');
+import users from './server/routes/users';
+import questions from './server/routes/questions';
+import rsvp from './server/routes/rsvp';
+import meetups from './server/routes/meetups';
 
 const app = express();
-
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(helmet());
 app.use(compression());
-
-// Use Routes
 app.use('/api/v1/users', users);
```

Comment:
Well done @danwebguy. Please ensure to move line 17 - 20  to route folder.

Reply:
ok @tosmak16